tests/utils.go

```diff
@@ -2910,14 +2902,14 @@ func configureIPv6OnVMI(vmi *v1.VirtualMachineInstance, expecter expect.Expecter
 		&expect.BExp{R: prompt},
 		&expect.BSnd{S: "echo $?\n"},
 		&expect.BExp{R: "0"},
-		&expect.BSnd{S: fmt.Sprintf(`echo "nameserver %s" >> /etc/resolv.conf\n`, dnsServerIP)},
-		&expect.BExp{R: prompt},
-		&expect.BSnd{S: "echo $?\n"},
-		&expect.BExp{R: "0"}})
+		&expect.BSnd{S: "ip -6 r | grep via | grep src\n"},
+		&expect.BExp{R: "default via fd10:0:2::1"},
+		&expect.BSnd{S: "sync\n"},
```

Comment:
```
{"component":"tests","kind":"VirtualMachineInstance","level":"info","msg":"Configure ipv6 GOOD: [{1 \r\n$  [$ ]} {3 \r\n$  [$ ]} {5 sudo ip -6 addr add fd10 [0]} {7 :0:2::2/120 dev eth0\r\n$  [$ ]} {9 echo $?\r\n0\r\n$  [$ ]} {11 sleep 5\r\n$ sudo ip -6 route add default via fd10 [0]} {13 :0:2::1 src fd10:0:2::2\r\n$ echo $?\r\n0\r\n$ ip -6 r | grep via | grep src\r\ndefault via fd10:0:2::1  [default via fd10:0:2::1]} {15 dev eth0  src fd10:0:2::2  metric 1024 \r\n$  [$ ]}]","name":"testvmifkpxfvnmvrv7pd6lssnfwhxhxp6fxdvpw2b98m5qt8xfg5mp","namespace":"kubevirt-test-default","pos":"utils.go:2918","timestamp":"2020-06-11T14:13:00.468152Z","uid":"88bbb828-23ef-4607-8bb1-c4d0aef0ded4"}
```
FYI 
i added the sync so there wont be consumable characters left
i could wait for 
default via fd10:0:2::1 dev eth0  src fd10:0:2::2  metric 1024
but didnt want a static metric, if it will be changed one day it would fail

i could wait just for prompt, but wanted to be safer

Reply:
imo sudo su and exit would be the best, because it will sandbox all stdout and stderr